lib/helpers/versions.js

```diff
@@ -96,3 +93,20 @@ function isPastLatest(dep, version) {
   logger.warn(`No dist-tags.latest for ${dep.name}`);
   return false;
 }
+
+function getRange(version) {
+  const parsedRange = semverUtils.parseRange(version);
+
+  // Check whether the existing version range is _simple_, meaning
+  // the range is a single operator, like `^`, `~`, `>=`, etc
+  return parsedRange.length === 1 && parsedRange[0].operator !== undefined ?
+
+    // If the project is using a _simple_ version range, lets use that range
+    // for the upgraded minimum version
+    parsedRange[0].operator :
+
+    // If no operator exists then return an empty string.
+    // This will cause the dependency version to get pinned
+    // to a specific version, which is the existing behavior of `renovate`
```

Comment:
@rarkins in the case where the version is either pinned, or using advance semantic range semantics, then we keep `renovate`'s existing behavior.

Only when the semantic version range is _simple_ do we keep the range.

Therefore applications can, if they choose, pin dependencies in their `package.json` file and this change will respect that.